### PR TITLE
Update paths

### PR DESCRIPTION
**Pull request by piersy**
Fixes paths such as /path/to/old/datadir/chaindata to be /path/to/old/datadir/celo/chaindata

Changed the name of environment variable OP_GETH_DIR to OP_GETH_DATADIR to make it more clear that it is the datadir.

Avoided use of the path `/celo` inside docker containers for the celo datadir, since it could easily be confused with the child celo dir under the celo datadir (I used /datadir instead).

Updated the paths for the migration to keep all the files under /migration-files

Removed (I think) unnecessary environment variable setting when running op-geth in docker '-e GETH_DATADIR=/celo'

### CODE REVIEW

File: docs/cel2/l2-operator-guide.md [hunk 1]
```diff
@@ -368,16 +368,15 @@ make geth
...
 # Copying the jwt secret from the op-node directory
-cp ${OP_NODE_DIR}/jwt.txt ${OP_GETH_DIR}/jwt.txt
+cp ${OP_NODE_DIR}/jwt.txt ${OP_GETH_DATADIR}/jwt.txt
 
 docker run -d \
   --name op-geth \
   --network=host \
-  -v ${OP_GETH_DIR}:/celo \
-  -e GETH_DATADIR=/celo \
+  -v ${OP_GETH_DATADIR}:/datadir \
```
Comment (piersy) [inline docs/cel2/l2-operator-guide.md:376]: @jcortejoso can you verify if this is a legitimate change?

**Review by jcortejoso — APPROVED** [review]

File: docs/cel2/l2-operator-guide.md (hunk 1 above)
Comment (jcortejoso) [reply docs/cel2/l2-operator-guide.md:376]: `GETH_DATADIR` helps as you don't need to carry over the `--datadir` flag for any other command. Especially convenient for `geth attach`

**Comment by jcortejoso** [comment]
Also something a noticed is folders under `/tmp` may be problematic for MacOS environments, or at least I could manage to get Docker for Mac volume sharing working with folder under `/tmp` (as /tmp is really a virtual folder under private/tmp).

It may help if we change to some folder under `$HOME` just as safety for some docker corner cases.

**Review by sodofi — APPROVED** [review]

File: docs/cel2/l2-operator-guide.md (hunk 1 above)
Comment (piersy) [reply docs/cel2/l2-operator-guide.md:376]: Ahh ok got you, hadn't thought about that.

**Comment by piersy** [comment]
Hi @jcortejoso I was also considering to go for $HOME but for a different reason that if you put files under /tmp you lose them when you restart, which could be problematic for people. Anyway this is merged now, but we can do another PR for that.

**Pull request merged**